src/diffusers/models/embeddings/image_text.py

```diff
@@ -0,0 +1,630 @@
+from typing import List, Tuple, Union
+
+import numpy as np
+import torch
+import torch.nn as nn
+
+from ...utils import deprecate
+from ..activations import FP32SiLU
+
+
+def get_2d_sincos_pos_embed(
```

Comment:
Could have keep these inside `position.py` but they are only used by `PatchEmbed` which technically isn't about just position embedding. Moreover, we apply `PatchEmbed` primarily when dealing with image and image-like data. 

So, it made sense.

Reply:
let's put patch and positioning embedding in the same file
and keep `image_text` mostly for simple projections

Reply:
> let's put patch and positioning embedding in the same file

Do you mean to keep them under `position.py`?